lib/src/server/context_registry.dart

```diff
@@ -124,8 +126,17 @@ class _ClientContextImpl implements ClientContext {
   final String traceId;
 
   @override
-  bool get isDevelopmentEnvironment => applicationContext.isDevelopmentEnvironment;
+  bool get isDevelopmentEnvironment =>
+      applicationContext.isDevelopmentEnvironment;
 
   @override
   bool get isProductionEnvironment => !isDevelopmentEnvironment;
 }
+
+String _headerOrEmptyString(HttpHeaders headers, String headerName) {
+  final elements = headers[headerName];
```

Comment:
I'm afraid `elements` can be `null` if there is no header named `headerName`.

The implementation is like this:
```
  List<String> operator [](String name) => _headers[_validateField(name)];
```
_headers is a `Map`.